code/game/objects/structures/false_walls.dm

```diff
@@ -48,6 +48,13 @@
 			update_icon()
 	opening = 0
 
+/obj/structure/falsewall/attack_animal(mob/living/simple_animal/user)
+	if(user.environment_smash)
+		user.do_attack_animation(src)
+		playsound(src.loc, 'sound/weapons/Genhit.ogg', 50, 1)
```

Comment:
probably more appropriate to use the same sound that wall smashing uses: `playsound(src, 'sound/effects/meteorimpact.ogg', 100, 1)`